src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -109,41 +133,35 @@ public override ExportResult Export(in Batch<Metric> batch)
 
                         if (exponentialHistogramData.ZeroCount != 0)
                         {
+#if NET
+                            bucketsBuilder.AppendLine(CultureInfo.InvariantCulture, $"Zero Bucket:{exponentialHistogramData.ZeroCount}");
+#else
                             bucketsBuilder.AppendLine($"Zero Bucket:{exponentialHistogramData.ZeroCount}");
+#endif
                         }
 
                         var offset = exponentialHistogramData.PositiveBuckets.Offset;
                         foreach (var bucketCount in exponentialHistogramData.PositiveBuckets)
                         {
                             var lowerBound = Base2ExponentialBucketHistogramHelper.CalculateLowerBoundary(offset, scale).ToString(CultureInfo.InvariantCulture);
                             var upperBound = Base2ExponentialBucketHistogramHelper.CalculateLowerBoundary(++offset, scale).ToString(CultureInfo.InvariantCulture);
+#if NET
+                            bucketsBuilder.AppendLine(CultureInfo.InvariantCulture, $"({lowerBound}, {upperBound}]:{bucketCount}");
+#else
                             bucketsBuilder.AppendLine($"({lowerBound}, {upperBound}]:{bucketCount}");
+#endif
                         }
                     }
 
                     valueDisplay = bucketsBuilder.ToString();
                 }
                 else if (metricType.IsDouble())
                 {
-                    if (metricType.IsSum())
-                    {
-                        valueDisplay = metricPoint.GetSumDouble().ToString(CultureInfo.InvariantCulture);
-                    }
-                    else
-                    {
-                        valueDisplay = metricPoint.GetGaugeLastValueDouble().ToString(CultureInfo.InvariantCulture);
-                    }
+                    valueDisplay = metricType.IsSum() ? metricPoint.GetSumDouble().ToString(CultureInfo.InvariantCulture) : metricPoint.GetGaugeLastValueDouble().ToString(CultureInfo.InvariantCulture);
```

Comment:
nit: I think we could separate this on additional lines to make it more readable.
But I won't block the PR on this. :)

```suggestion
                    valueDisplay = metricType.IsSum() 
                        ? metricPoint.GetSumDouble().ToString(CultureInfo.InvariantCulture)
                        : metricPoint.GetGaugeLastValueDouble().ToString(CultureInfo.InvariantCulture);
```